app/Providers/SpotifyServiceProvider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\ServiceProvider;
+use SpotifyWebAPI;
+
+class SpotifyServiceProvider extends ServiceProvider
+{
+    /**
+     * Register services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        $this->app->bind(SpotifyWebAPI\Session::class, function ($app) {
+            return new SpotifyWebAPI\Session(
+                config('spotify.client_id'),
+                config('spotify.client_secret'),
+                'http://festival-planner.test/spoti'
```

Comment:
```suggestion
                url('/spoti')
```
This url should be dynamic as the the url of you application can change.